bootstrap/kubeadm/docs/external-etcd.md

```diff
@@ -11,13 +11,32 @@ Before getting started you should be aware of the expectations that come with us
     * As an example, cross availability zone traffic can cost money on cloud providers. You don't have to deploy etcd
     across availability zones, but if you do please be aware of the costs.
 
-### Getting started
+## Getting started
 
 To use this, you will need to create an etcd cluster and generate an apiserver-etcd-client key/pair.
-[`etcdadm`](https://github.com/kubernetes-sigs/etcdadm) is a good way to get started if you'd like to test this
-behavior.
+This behaviour can be tested using [`kubeadm`](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) and [`etcdadm`](https://github.com/kubernetes-sigs/etcdadm).
 
-Once you create an etcd cluster, you will want to base64 encode the `/etc/etcd/pki/apiserver-etcd-client.crt`,
+
+### Using kubeadm 
+Once you created the etcd cluster, CA certificates are required.
+If you already have a CA then the CA's `key` and `crt` must be copied to `/etc/kubernetes/pki/etcd/ca.crt` and `/etc/kubernetes/pki/etcd/ca.key`.
+
+If you do not already have a CA then run command `kubeadm init phase certs etcd-ca`. This creates two files
+
+ * `/etc/kubernetes/pki/etcd/ca.crt`
+ * `/etc/kubernetes/pki/etcd/ca.key`  
+
+### Using etcdadm
```

Comment:
are we OK with promoting etcdadm given it's still pre-alpha?

Reply:
although i don't really know what the actual state is, i've heard "pre-alpha" from someone in a meeting, but i don't remember who. @dlipovetsky are you OK with this documentation addition?

Reply:
We should probably add some sort of big banner like we do in the quick start documentation stating that etcdadm is still a work in progress, or remove it from here

Reply:
I'm for having the doc, but with a disclaimer and perhaps a friendly "help wanted" :smile: